Formula/libmpdec.rb

```diff
@@ -0,0 +1,52 @@
+class Libmpdec < Formula
+  desc "Library for decimal floating point arithmetic"
+  homepage "https://www.bytereef.org/mpdecimal/"
+  url "https://www.bytereef.org/software/mpdecimal/releases/mpdecimal-2.4.2.tar.gz"
+  sha256 "83c628b90f009470981cf084c5418329c88b19835d8af3691b930afccb7d79c7"
+
+  patch do
```

Comment:
This makes `brew audit --new-formula libmpdec` return an error.
I've tried to contact the library author to have the patch applied upstream but no reply yet.